lib/google/longrunning/operations_client.rb

```diff
@@ -172,14 +136,26 @@ def initialize \
       #   name = ''
       #   response = operations_client.get_operation(name)
 
-      def get_operation \
-          name,
-          options: nil
-        req = {
-          name: name
-        }.delete_if { |_, v| v.nil? }
-        req = Google::Gax.to_proto req, Google::Longrunning::GetOperationRequest
-        @get_operation.call req, options
+      def get_operation request = nil, options: nil, **request_fields, &block
+        raise ArgumentError, "request must be provided" if request.nil? && request_fields.empty?
+        if !request.nil? && !request_fields.empty?
+          raise ArgumentError, "cannot pass both request object and named arguments"
+        end
+
+        request ||= request_fields
+        # request = Google::Gax::Protobuf.coerce request, to: Google::Showcase::V1alpha3::EchoRequest
+        request = Google::Gax.to_proto request, Google::Longrunning::GetOperationRequest
+
+        # Converts hash and nil to an options object
+        options = Google::Gax::ApiCall::Options.new options.to_h if options.respond_to? :to_h
+        echo_header_params = { name: request.name }
+        echo_request_params_header = echo_header_params.map { |k, v| "#{k}=#{v}" }.join("&")
+        echo_metadata = @metadata.merge "x-goog-request-params" => echo_request_params_header
+        echo_retry_policy = { retry_codes: [GRPC::Core::StatusCodes::UNAVAILABLE] }
```

Comment:
This is probably safe to do in LRO methods, but it might be wise to open an issue to verify this (if we don't already have one - I know I bring this up a lot).

Reply:
Removed.